pages/dev.md

```diff
@@ -1,3 +1,5 @@
+# Development
+
```

Comment:
Ah, I was cutting these out and replacing them with a title: attribute since the title is already displayed at the top of the article and it looked weird to duplicate (even if the top display is in smaller text)